tests/integration/conftest.py

```diff
@@ -368,9 +370,8 @@ async def app_fixture(
 
     yield app
 
-    # logger.info("Removing application.")
-    # await test_configs.model.remove_application(app_name=app.name, force=True, no_wait=True)
-    # logger.info("Application removed.")
+    # Do not clean up due to Juju bug in model.remove_application. However, manual cleanup is
+    # required on private-endpoint OpenStack resources.
```

Comment:
Do we have a ticket or something to follow up on? And a guidance where/howto do this internally (e.g. if you are not available). This is likely to cause problems because multiple engineers are relying on the availability of private-endpoint OpenStack resources and might run into quota issues in their tests.

Reply:
I intentionally left that part out because that is heavily related to our OpenStack deployment. I can note it down on our internal GitHub runner model management documentation.